### PR TITLE
Image placement policy module and its info module 

### PR DESCRIPTION
**Pull request by bhati-pradeep**
New modules:
- ntnx_image_placement_policy
- ntnx_image_placement_policies_info

### CODE REVIEW

**Review by Gevorg-Khachatryan-97 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -110,6 +110,8 @@ ansible-playbook examples/iaas/iaas.yml
 | ntnx_floating_ips_info | List existing Floating_Ips. |
 | ntnx_pbrs | Create or delete a PBR. |
 | ntnx_pbrs_info | List existing PBRs. |
+| ntnx_image_placement_policy | Create, update or delete a image placement policy. |
```
Comment (premkarat) [inline README.md:113]: can you order them in ascending name order

File: plugins/modules/ntnx_image_placement_policies_info.py [hunk 2]
```diff
@@ -0,0 +1,289 @@
...
+
+
+def get_policy(module, result):
+    policy_obj = ImagePlacementPolicy(module)
+    uuid = module.params.get("policy_uuid")
+    resp = policy_obj.read(uuid)
+    result["response"] = resp
+
+
+def get_particular_policies(module, result):
```
Comment (premkarat) [inline plugins/modules/ntnx_image_placement_policies_info.py:253]: use word specific instead of particular

Comment (premkarat) [reply plugins/modules/ntnx_image_placement_policies_info.py:253]: should it be get_all_policies()

**Review by premkarat — CHANGES_REQUESTED** [review]

**Pull request merged**